instance/models/openedx_appserver.py

```diff
@@ -208,6 +208,7 @@ def create_configuration_settings(self):
             # querying the Github API twice), we pass it as a context variable.
             'github_admin_username_list': self.github_admin_username_list,
             'newrelic_license_key': settings.NEWRELIC_LICENSE_KEY,
+            'smtp_relay_settings': self.smtp_relay_settings
```

Comment:
@mtyaka That's a great solution! Makes the code in `create_configuration_settings` much nicer, I like it :)